src/main/resources/META-INF/rewrite/swagger-2.yml

```diff
@@ -256,20 +256,3 @@ recipeList:
       annotationType: io.swagger.v3.oas.annotations.media.Schema
       newAttributeName: allowEmptyValue
       oldAttributeName: nullable
-
----
-type: specs.openrewrite.org/v1beta/recipe
-name: org.openrewrite.openapi.swagger.MigrateApiModelToSchema
```

Comment:
Mark as done if already verified: Is this not used in a composite recipe anywhere?